docs/community/local-groups.mdx

```diff
@@ -26,6 +26,8 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Laguna Mesh](https://lagunamesh.com)
 - [Mission Viejo Mesh](https://missionviejomesh.org/)
 - [Bay Area Mesh](https://canvis.app/z2k8a8)
+- [San Diego Mesh]
+(https://discord.gg/HzE248x2)
```

Comment:
This needs to be on the same line as `[San Diego Mesh]` with no space between

Reply:
I pushed a commit to fix